@JRC/bootstrap.m

```diff
@@ -146,7 +162,15 @@ function bootstrap(obj, varargin)
         break;
     end
 
-    dlgAns = questdlg('Would you like to export advanced parameters?', 'Bootstrap', 'No');
+    if ask 
+        dlgAns = questdlg('Would you like to export advanced parameters?', 'Bootstrap', 'No');
+    else
```

Comment:
We can drop a level of indentation by doing

```matlab
if ask 
    dlgAns = questdlg('Would you like to export advanced parameters?', 'Bootstrap', 'No');
elseif advanced
    dlgAns = 'Yes';
else
   dlgAns = 'No';
end

Reply:
True enough